go/kbfs/libkbfs/test_common.go

```diff
@@ -808,7 +808,11 @@ func ForceQuotaReclamationForTesting(config Config,
 // test if there's an error.
 func CheckConfigAndShutdown(
 	ctx context.Context, t logger.TestLogBackend, config Config) {
-	if err := config.Shutdown(ctx); err != nil {
+	err := config.Shutdown(ctx)
+	switch err.(type) {
```

Comment:
`errors.Cause(err)` just in case?

Reply:
Hmm, yeah. I'll add that in the next one.